Core/include/Acts/Propagator/Propagator.hpp

```diff
@@ -178,6 +180,52 @@ struct PropagatorOptions : public PropagatorPlainOptions {
   std::reference_wrapper<const MagneticFieldContext> magFieldContext;
 };
 
+/// Common simplified base interface for propagators.
+///
+/// This class only supports propagation from start bound parameters to a target
+/// surface and returns only the end bound parameters.
+/// Navigation is performed if the underlying propagator is configured with an
+/// appropriate navigator. No custom actors or aborters are supported.
+class BasePropagator {
+ public:
+  /// Base propagator options
+  using Options = PropagatorOptions<>;
```

Comment:
curvilinear overload does not make sense here because we locked down the options right?

Reply:
You mean curvilinear start params?

Reply:
like here with final curvilinear params

https://github.com/acts-project/acts/blob/cbd38cd023e71a189b9845c4932c104f12cb7b28/Core/include/Acts/Propagator/Propagator.hpp#L374-L380

Reply:
Added a line to the test which executes this. So I think this works.